basic.py

```diff
@@ -90,3 +91,33 @@ def eval_(event):
         return eval(input, {"__builtins__": None}, safe_dict)
     except Exception as e:
         return "eval: {0}".format(e)
+
+
+last_events = {}
+
+
+@event
+def sub_event_catcher(event):
+    if event.event_type in ['message', 'cmd']:
+        last_events[event.meta['host']] = event
+
+
+@match(regex=r'^s/(.*?)/(.*?)/([a-z]*)$')
```

Comment:
@mrshu what if we want to use `/` inside the matched string? Wouldn't it be better to do it just like `sed` does?